src/plugins/workspace/server/types.ts

```diff
@@ -128,6 +129,9 @@ export interface AuthInfo {
   user_name?: string;
 }
 
+export interface AppPluginSetupDependencies {
+  applicationConfig: ApplicationConfigPluginSetup;
```

Comment:
```suggestion
  applicationConfig?: ApplicationConfigPluginSetup;
```
applicationConfig is an optional dependency of workspace.